homework/06LessonSix.js

```diff
@@ -96,7 +106,7 @@ var conferenceRoom = new ConferenceRoom();
 // Write your answer in comments
 //
 // What is a method?
-//
+// A method is a function within an object.
```

Comment:
Yes, so invoking method on object means we are doing something to the object via that method